<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectEditor</class>
 <widget class="QWidget" name="ObjectEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item alignment="Qt::AlignTop">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignLeft">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="objectSave">
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/rc/icons/checkmark.svg</normalon>
            </iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QWidget" name="horizontalWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>95</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGraphicsView" name="objectSpritePreview">
           <property name="maximumSize">
            <size>
             <width>86</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Object Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="objectNameEdit">
             <property name="text">
              <string>object_name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Default Sprite</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="objectDefaultSpriteDropdown"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="objectPersistent">
          <property name="text">
           <string>Persistent</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1000000</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="objectCodeEditor">
     <property name="plainText">
      <string>-- initialisation code here

function update()
    -- code here
end

function draw()
    -- code here
end</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
